_data/beps_completed.yml

```
@@ -13,13 +13,33 @@
     - name: Kirstie Whitaker
   content:
     - raw
+  google_doc_created: 2017-02
+  pull_request_created: 2020-06
+  pull_request_merged: 2021-02
+
+- number: "002"
+  title: BIDS Models Specification
+  leads:
+    - name: Tal Yarkoni
+    - name: Chris Markiewcz
+  content:
+    - file format
+  outcome: "Became an [independent specification with validator](https://bids-standard.github.io/stats-models/) to execute statistical models, with minimal configuration and intervention required from the user."
+  google_doc_created: 2016-09
+  pull_request_created: 2018-10
+  pull_request_merged: 2023-08
 
 - number: "003"
   title: Common Derivatives
   leads:
     - name: Chris Markiewicz
+  former_leads:
+    - name: Chris Gorgolewski
   content:
     - derivative
+  google_doc_created: 2018-10
```

The first doc has been around forever.

```suggestion
  google_doc_created: 2016-02
```